openemory/publication/templates/publication/site_index.html

```diff
@@ -63,13 +64,16 @@
             <div id="wrapper">
               <div id="slides">
                 <div class="slides_container">
-                  <a href="{{ sitepages.about.url }}" title="Welcome">
+                  {% get_flatpages '/about/' as about_page %}
+                  <a href="{{ about_page.first.url }}" title="Welcome">
                     <img src="{{ STATIC_URL }}images/slides/OE_SliderGuide_welcome.jpg" width="705" height="225" alt="Slide 1"/>
                   </a>
-                  <a href="{{ sitepages.getting_started.url}}" title="Welcome">
+                  {% get_flatpages '/about/getting-started/' as start_page %}
```

Comment:
https://open.library.emory.edu/about/getting-started/ is a 404 at the moment. Is that intentional?